### PR TITLE
added redis config

### PR DESCRIPTION
**Pull request by cristianciutea**

### CODE REVIEW

File: package_repo_update/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+- name: Update yum
+  command: yum update
```
Comment (rogercoll) [inline package_repo_update/tasks/main.yaml:3]: Is there any reason why we are not the builtin ansible yum/apt modules?

If so, could we add the `command-instead-of-module` to skip specific rules in the ansible linter pipeline? https://github.com/newrelic-experimental/caos-ansible-roles/runs/5582525367?check_suite_focus=true#step:4:16

Comment (rogercoll) [reply package_repo_update/tasks/main.yaml:3]: `yum update` also tries to upgrade the packages if any is eligible. And asks for the approval (yes/no question)

If we only want to update packages indexes I would change it for `yum check-update`.

**Review by rubenruizdegauna — APPROVED** [review]
LGTM

**Review by rogercoll — APPROVED** [review]
LGTM

**Pull request merged**